indico/web/flask/util.py

```diff
@@ -257,15 +257,14 @@ def url_for(endpoint, *targets, **values):
         values.update(locator)
 
     static_site_mode = bool(ContextManager.get('offlineMode'))
-    values.setdefault('_external', static_site_mode)
```

Comment:
are you sure this is correct? IIRC i added it so links that cannot be made available offline (registration, surveys) will have absolute links pointing to the live version